### ISSUE TITLE
HIGH: edge_lifecycle view — realized_pnl sign-inverted for actual_outcome='no' (fake gains on lost positions)

### ISSUE THREAD

**Issue by mutantamoeba** [mutantamoeba/precog#909]
## Origin

Pre-S68 backlog audit, session 65 (\`memory/audit_pre_s68_backlog.md\`). Finding originally raised by claude-review on PR #436 in the no-read-comments era and never acted on. **89% of \`### Bug:\` headers in pre-S68 claude-reviews were live-rate on audit.**

## Summary

The \`edge_lifecycle\` view's \`realized_pnl\` formula is sign-inverted for the \`actual_outcome = 'no'\` branch. Analytics queries reading \`edge_lifecycle.realized_pnl\` for NO outcomes **report fake gains on lost YES positions.**

## Evidence (verified on main)

\`src/precog/database/alembic/versions/0058_business_key_fk_renames.py:344\` (latest view definition, still has the bug):
\`\`\`sql
CASE
  WHEN actual_outcome = 'yes' THEN (settlement_value - market_price) * position_size
  WHEN actual_outcome = 'no'  THEN (market_price - settlement_value) * position_size  -- INVERTED
  ELSE NULL
END AS realized_pnl
\`\`\`

Also present at:
- \`0023_edges_enrichment.py:151\` (original definition)
- \`0024_enum_to_varchar.py:299\` (view re-creation)
- \`0058_business_key_fk_renames.py:477\` (downgrade path — must also fix)

## Correct formula

For a YES-side position (edges detect buy-YES opportunities — see PR #436 context comment at \`0023_edges_enrichment.py:148\` \`-- P&L assumes YES-side position\`):

- YES outcome: gain = settlement_value (1.0) − market_price_paid
- NO outcome: loss = 0.0 − market_price_paid = −market_price_paid (**negative**, not positive)

The fix: **both branches** use \`(settlement_value - market_price) * position_size\`. The sign handles the loss naturally — \`settlement_value = 0\` for lost NO outcomes produces a negative P&L.

## Why this matters

- **Money-path correctness.** Any consumer of \`edge_lifecycle.realized_pnl\` — reporting dashboards, strategy ROI calculations, the \`update_strategy_metrics\` / \`update_model_metrics\` paths that track \`paper_roi\` / \`live_roi\` — will overstate performance on losing NO outcomes.
- **Gates Phase 2 manual trading.** #504 manual trade placement epic cannot safely ship while this bug is live. Fake gains in the dashboard on losing positions is a user-visible correctness failure.
- **Pattern signal:** the bug sits directly under the code comment \`-- P&L assumes YES-side position (edge detection = buy YES)\`. The "assumes" language is a red flag — the code should have enforced the assumption structurally, not relied on every future reader to get the math right.

## Proposed fix

1. New migration \`0065_fix_edge_lifecycle_realized_pnl.py\` with:
   - \`CREATE OR REPLACE VIEW edge_lifecycle\` using corrected formula (both branches use \`settlement_value - market_price\`)
   - Downgrade path restores the (inverted) previous definition for reversibility
2. Update \`testcontainers_fixtures.py\` if it mirrors the view definition
3. Unit test exercising the view with a losing NO outcome + asserting negative \`realized_pnl\`
4. Grep for any Python code that duplicates the formula independently (there should be none, but verify)

## Priority

**HIGH.** Gates Phase 2 trade-execution launch. Not a migration-time correctness issue (0058 migration doesn't emit data), but every downstream consumer of the view is wrong for NO outcomes.

## Suggested dispatch

Full Tier 2 pipeline (Samwise Builder, Glokta Reviewer — money-path adversarial, Ripley Sentinel). Holden for migration safety review. Money-touching so Reviewer MUST be Glokta not Spock.
